pages/common/virt-clone.md

```diff
@@ -0,0 +1,8 @@
+# virt-clone
+
+> Clone a libvirt virtual machine.
+> More information: <https://virt-manager.org/>.
+
+- Clone a virtual machine generating a new virtual machine name and storage path from the original name and path, and a random MAC address:
```

Comment:
```suggestion
- Clone a virtual machine with a random MAC address:
```
> generating a new virtual machine name and storage path from the original name and path

Isn't that already clear, since this is what cloning does? Please correct me if I'm wrong.

Reply:
I think I've addressed this in my latest commit.